### ISSUE TITLE
GameGuru MAX - BUG - Particle Opacity Less Than 5 Reverts to 0

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#2152]
You can't set an Opacity less than 5 for Particles as it will set it automatically to 0. 5 and up is fine, though.

**Comment by LeeBamberTGC** [comment]
@dllllllllllb A nice one for you to investigate. First find out why the value is changed through the conversion into storage, and then provide an idea for the solution. We can then discuss here.

**Comment by dllllllllllb** [comment]
@LeeBamberTGC Looked into it, found that there is a rounding issue with other numbers as well (e.g. 11 was getting changed to 10) and that the issue persists on other sliders as well. 

I thought the issue is with c-style casting, the order of operations or maybe conversion made by ImGui , but it turns out that the denominator values were missing f at the end to indicate that the value is divided by float. Example.:
tmpint / 100.0 <- Doesn't work
tmpint / 100.0f <- Works

I'll try to find all the sliders with this issue but it might take a while

**Comment by dllllllllllb** [comment]
@LeeBamberTGC There is also an issue where even though the slider says that the particle opacity is 0, you can still see very faint particles. Changing the v_min slider value from 1 to 0 fixes this, should I also change these?

**Comment by LeeBamberTGC** [comment]
@dllllllllllb Yes indeed, use your best judgement on things like this. What makes sense to you will highly likely make sense to most users. Zero should mean cannot see it at all :)

**Comment by dllllllllllb** [comment]
@LeeBamberTGC I just wanted to check if there is a reason for the min_val to be 1, don't want to change it in case it's used as a denominator somewhere :]

**Comment by LeeBamberTGC** [comment]
@dllllllllllb Probably a quick typo. Set to zero, and commit, we can then see if all is well and particle opacity of zero hides them.
